cypress/integration/contracts/erc20.spec.ts

```diff
@@ -39,8 +39,6 @@ describe('ERC20 Contract ', () => {
       .and('contain', 'balances:Reserved')
       .and('contain', 'balances:Withdraw')
       .and('contain', 'system:ExtrinsicSuccess');
-
-    cy.get('[data-cy="dismiss-notification"]').click();
```

Comment:
why did you have to change the test file? 
the tests check for UI notifications so it's important to close them between message calls. at the moment doesn't make a difference because we don't test multiple transactions in the same test, but that will change very soon.

Reply:
Don't remember now. Removed